Olfeo/_meta/manifest.yml

```
@@ -0,0 +1,5 @@
+uuid: f74ed0e8-a4e2-43e3-9bfc-a846c2c7d612
+name: Olfeo secure web gateway
+slug: olfeo
+description: >-
+        Olfeo Secure Web Gateway is a suite of cybersecurity features for analyzing, filtering and securing your web flows. Combining proxy filtering, flow antivirus and DNS filtering.
```

The same for the description:
```suggestion
        Olfeo is a leading provider of web security solutions, offering customizable web filtering and secure web gateway services to businesses worldwide. Their technology helps organizations protect against online threats and enforce internet usage policies effectively.
```